recipes/metawrap/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "metawrap" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/bxlab/metaWRAP/archive/v{{ version }}.tar.gz
+  md5: face9197bb696b037d7b2acdf27c668a
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build: 
+
+  run:
+    - bcftools 1.6
```

Comment:
we should not pin all these packages so strict imho.

Reply:
This is a good question. This is due to the high number of dependencies for the software. I cannot relax too many software dependencies because then the conda installation takes too long, as there are too many permutations that conda has to recursively consider when building the environment. See issue https://github.com/bxlab/metaWRAP/issues/74. I tried more relaxed settings previously but the Bioconda test build step just timed out. Additionally, I want to keep the dependencies somewhat strict to make it future-proof. Because of the number of pieces working together, it is likely that any change in them may break the pipeline.